pkg/controller/migmigration/task.go

```diff
@@ -1,29 +1,44 @@
 package migmigration
 
 import (
+	"context"
 	"fmt"
 	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
 	"github.com/go-logr/logr"
 	velero "github.com/heptio/velero/pkg/apis/velero/v1"
+	corev1 "k8s.io/api/core/v1"
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 // Annotation Keys
-const MigQuiesceAnnotationKey = "openshift.io/migrate-quiesce-pods"
+const (
+	MigQuiesceAnnotationKey        = "openshift.io/migrate-quiesce-pods"
+	copyBackupRestoreAnnotationKey = "openshift.io/copy-backup-restore"
```

Comment:
Is there a reason one annotation name is exported but the other is not?

Reply:
I don't remember why we exported the quiesce annotation. I think we should not export the quiesce annotation, it's only used in this package